tillerinobot/src/main/java/tillerino/tillerinobot/lang/Greek.java

```diff
@@ -0,0 +1,215 @@
+package tillerino.tillerinobot.lang;
+
+import java.util.List;
+import java.util.Random;
+
+import org.tillerino.osuApiModel.Mods;
+import org.tillerino.osuApiModel.OsuApiUser;
+
+import tillerino.tillerinobot.BeatmapMeta;
+import tillerino.tillerinobot.IRCBot.IRCBotUser;
+import tillerino.tillerinobot.RecommendationsManager.Recommendation;
+
+public class Default implements Language {
+
+	@Override
+	public String unknownBeatmap() {
+		return "Συγνώμη, δεν γνωρίζω αυτό το τραγούδι. Ίσως είναι αρκετά νεο, πολυ δύσκολο, μη εγκεκριμένο ή να μην είναι για το osu standard mode." ;
+	}
+
+	@Override
+	public String internalException(String marker) {
+		return "Εχ... μάλλον φαίνεται ότι ο ανθρώπινος Tillerino έκανε μαντάρα την σύνδεσή μου."
+				+" Εάν δεν το παρατηρήσει σύντομα, μπορείς [https://github.com/Tillerino/Tillerinobot/wiki/Contact να τον ενημερώσεις];" (reference "
```

Comment:
uh? What happened here (and in a lot of lines down there)? If you want to insert `"`, you need to do `\"` to make java  understand you want to print a `"`, not to make the string end.